gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -1292,6 +1322,24 @@ MM_ParallelGlobalGC::fixHeapForWalk(MM_EnvironmentBase *env, UDATA walkFlags, ui
 	return fixedObjectCount;
 }
 
+uintptr_t
+MM_ParallelGlobalGC::clearHeap(MM_EnvironmentBase *env,UDATA walkFlags, uintptr_t walkReason, MM_HeapWalkerObjectFunc walkFunction)
```

Comment:
This method actually does not need walkReason parameter - it can be just hardcoded within the method.
Same for walkFlags.